JULIERAJ/KIDS-FIRST-Front
# Change password page and Changed password confirmation

Change password page and Changed password confirmation

## One Line Description
This pull request addresses the tickets #46 and #36 for password change pages in frontend.

## Requirements
On password-reset page: 
- Adjust forget password and change password page to match the design in Figma.
- Display errors coming from the backend - not the default one.

On changed-password page:
- Adjust password changed page to match the design in Figma.
- Follow the best practices while accomplishing the task.
- Follow the measurements in Figma

## Notes
Given that changed password confirmation was displayed conditionally on  reset-password page, I completed the code change on the same branch as the reset password page change.

## Test Steps
1. Navigate to http://localhost:3000/forgot-password 
2. Provide email address for forgotten password. 
3. Click on 'Reset your password' in email received:
![image](https://github.com/user-attachments/assets/ddcfb8de-349d-4e8e-8be7-530cff689b1a)

4. Provide a new valid password on /reset-password page according to the  directions provided.
5. Click on Next.
6. Verify /changed-password page.
 
## UI/UX Outcome
[UI/UX Changes](https://app.screencastify.com/v3/watch/USXWB9Xx25GJMCJIHkLG)
